cmd/pFaTools/extractBed.go

```diff
@@ -0,0 +1,67 @@
+package main
+
+import (
+	"flag"
+	"fmt"
+	"github.com/vertgenlab/gonomics/bed"
+	"github.com/vertgenlab/gonomics/exception"
+	"github.com/vertgenlab/gonomics/fasta/pFasta"
+	"log"
+	"os"
+)
+
+// ExtractBedSettings defines the usage settings for the pFa extractBed subcommand.
+type ExtractBedSettings struct {
+	InFile  string
+	Region  string
+	OutFile string
+	TakeCoords bool
+}
+
+// extractBedUsage defines the usage statement for the pFaTools extractBed subcommand.
+func extractBedUsage(extractBedFlags *flag.FlagSet) {
+	fmt.Printf("pFaTools extractBed - a tool for extracting a subsequence from a pFa, as specified by a Bed file.\n" +
+		"A note on input and output file types:\n" +
+		"\tpFa (probabilistic FASTA) encodes position-wise probabilities in matrix entries. Columns should therefore sum to 1.\n" +
+		"\tbed (browser extensible data) stores genomic regions as coordinates and associated annotations." +
+		"Usage:\n" +
+		"pFaTools extractBed in.pFa region.bed outFile\n" +
```

Comment:
Can we add an extension to outfile, in this case outFile.pFa

Reply:
Added!